.gitignore

```diff
@@ -1,3 +1,5 @@
+.neuburg-backendrc
```

Comment:
This file should stay in the repository. Maybe put in in another directory like /templates/

Reply:
You mean that we put the "template" in the templates folder?  
Then `.neuburg-backenrc` would stay in the .gitignore right?

Reply:
Yes, I think that would make sense. Maybe also some docu about setting it up would make sense.